tee/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tee"
+version = "0.1.0"
+authors = ["Jeremy Jackson <git@jeremyvii.com>"]
+license = "MPL-2.0-no-copyleft-exception"
+build = "build.rs"
+edition = "2018"
+description = """
+Read from standard input and write to standard output or files.
+"""
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "^2.33.0", features = ["wrap_help"] }
+assert_cmd = "1.0.2"
+tempfile = "3.1.0"
```

Comment:
These are only used in tests, right? The it should be in `dev-dependencies` instead, so we don't have to compile those crates unless we are testing it